RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java

```diff
@@ -103,4 +104,29 @@ public void deleteStore(@PathVariable int id, HttpServletResponse response) thro
         }
     }
 
+    /**
+     * This method shows all products that sell at the current store
+     *
+     * @param id current store_id
+     * @param response
+     * @return list of products that sell at the current store
+     * @throws IOException if current store hasn't any product we inform user
+     */
+
+    @GetMapping("/{id}/products")
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.OK)
+    public List<Product> getAllProducts(@PathVariable int id, HttpServletResponse response) throws IOException {
```

Comment:
Please follwo the same style for annotations. If you are going to use @RequestMapping, please use it everywhere, If you @GetMapping, @Response status, then use those annotataions everywhere. Do not mix old and new style